src/editors/dateEditor.js

```diff
@@ -74,6 +74,7 @@ class DateEditor extends TextEditor {
    */
   destroyElements() {
     this.$datePicker.destroy();
+    this.datePicker.remove();
```

Comment:
`.remove()` method is not supported by IE (https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove)